universal-login-relayer/test/integration/sql/EmailConfirmationsStore.test.ts

```diff
@@ -0,0 +1,42 @@
+import {getKnexConfig} from '../../testhelpers/knex';
+import {EmailConfirmationsStore} from '../../../src/integration/sql/services/EmailConfirmationsStore';
+import {EmailConfirmation} from '@unilogin/commons';
+import {expect} from 'chai';
+
+describe('UNIT: EmailConfirmationsStore', () => {
+  const knex = getKnexConfig();
+  const emailConfirmationsStore = new EmailConfirmationsStore(knex);
+
+  it('Should add emailConfirmation to database and get it from it', async () => {
+    const storedEmail = 'newEmail@email.com';
+    const storedEmailConfirmation = {
+      email: storedEmail,
+      ensName: 'bob.unilogin.eth',
+      code: '1234',
```

Comment:
You can add `isConfirmed: false` here, and then you will have in the 20 line: `expect(await emailConfirmationsStore.get(email)).be.deep.eq(storedEmailConfirmation)`

Reply:
Done